### ISSUE TITLE
Email notifications or similar

### ISSUE THREAD

**Issue by wolftune** [GRA0007/crab.fit#135]
It would be ideal to have the capacity to get notified (probably by email) when responses are added or updated.

**Comment by GRA0007** [comment]
Yes! This is a feature I'd like too. Thanks for the suggestion :)
